README.md

```diff
@@ -143,17 +143,20 @@ There are several command line options to start the daemon:
 Currently you would likely want to start the daemon like this:
 
 ```bash
-cargo run -- -r http://knilxof.org:4242 --disable-tls
+./run.sh -- -r http://knilxof.org:4242 --disable-tls
```

Comment:
We discussed that with @julienw, we thought `--` still has some value, when you want to `--verbose` or `--freatures` on cargo, for instance.